ohno.md

```diff
@@ -124,24 +128,34 @@
 
 ... someone converts a turn restriction to a multipolygon.
 
+... someone converts a multipolygon to a turn restriction.
+
 ... iD gets stuck in one position.
 
 ... one of your changesets is downvoted in OSMCha.
 
 ... someone forgets to add turn restrictions.
 
+... someone adds unnecessary turn restrictions.
+
 ... a node is moved halfway around the world.
 
+... a node is moved by less than a millimeter.
+
 ... someone adds redundant nodes to a straight way.
 
+... someone doesn't add enough nodes to a curved way.
+
```

Comment:
This isn't exactly an issue, it's just that they mapped the thing in lowe-quality.